static/includes/Reference/SCALEServicesSNMPFields.md.part

```diff
@@ -0,0 +1,26 @@
+**General Options**
+
+| Name | Description |
+|------|-------------|
+| Location | Enter the location of the system. |
+| Contact | E-mail address that receives SNMP service messages. |
+| Community | Change from *public* to increase system security. Can only contain alphanumeric characters, underscores (`_`), dashes (`-`), periods (`.`), and spaces. This can be left empty for SNMPv3 networks. |
+
+**SNMP v3 Options**
+
+| Name | Description |
+|------|-------------|
+| SNMP v3 Support | Set to to enable support for [SNMP version 3](https://tools.ietf.org/html/rfc3410). See [snmpd.conf(5)](https://net-snmp.sourceforge.io/docs/man/snmpd.conf.html) for configuration details. |
+| Username | Enter a username to register with this service. |
+| Authentication Type | Choose an authentication method: `---` for none, *[SHA](https://tools.ietf.org/html/rfc4634)*, or *[MD5](https://tools.ietf.org/html/rfc1321)*|
```

Comment:
I'm not sure why you used the ` ` tags unless that is something they type into the field. if it is on the dropdown list then use the bold tags. 
```suggestion
| Authentication Type | Choose an authentication method: `---` for none, **[SHA](https://tools.ietf.org/html/rfc4634)**, or **[MD5](https://tools.ietf.org/html/rfc1321)**|
```